### DISCUSSION TITLE
Has anyone ever made CurlFTPFs work with old Vax/VMS 7.3 systems?

### DISCUSSION THREAD

**Discussion by gtoal** [curl/curl#8957] [Q&A] [answered]
I'm running a virtual Vax/VMS under the simh simulator on a Pi4, and it's behind a networking config that means it only has access to machines internal to my home network.  The only networking the Vax/VMS build I'm using supports, is FTP and Telnet.  The FTP server  is an old one and there is no SSL involved at all.  Although in principle there is an alternative networking stack that could be installed, I'm not going to do that for various reasons - I need to live with what I've got in this image. So... as a bridge to outside users I've created a ssh->telnet gateway for interactive access, and I was hoping to create some sort of equivalent gateway for file transfers, by mounting the VMS on a gateway system using CurlFTPFs and making that mount available by other more secure means.
Unfortunately it appears that the format of 'ls' listings provided by the old Vax/VMS FTP server is incompatible with CurlFTPFs.   I doubt anyone else is interested in supporting a 40 year old system so I don't think it's worth submitting a bug report - I'll just try to fix it myself, but before I start hacking on it to read and parse what the Vax/VMS provides, I thought I'd ask if anyone else has done this before - perhaps a fork somewhere I haven't found?
Thank you,
Graham

**Comment by dfandrich** [comment, marked as answer]
Will you be hacking libcurl or curlftpfs? I suspect the latter, in which case
that's the project you want to be asking, not us.

**Reply by gtoal** [reply]
Sorry! My mistake - I followed a chain of links from the curlftpfs web page that ended up on this forum and didn't notice that somewhere in that chain I had switched projects! I thought I was posting on the curlftpfs site.
